firmware/quadruna/BMS/src/app/states/app_chargeState.c

```diff
@@ -138,7 +142,11 @@ static void chargeStateRunOnTick100Hz(void)
                                            ? app_canRx_Debug_ChargingVoltageTargetValue_get()
                                            : INITIAL_CHARGING_VOLTAGE;
 
-        if (IS_IN_RANGE(0.0f, MAX_CHARGING_VOLTAGE, charging_voltage))
+        const float max_charging_voltage = app_canRx_Debug_ChargingVoltageSafetyOverride_get()
```

Comment:
I might be missing something but in the CAN msg is only allocated 1 bit, but its cast to a float here. Should it be allocated 32 bits in the JSONCAN?

Reply:
The can message is just used as input to the conditional operator, max_charging_voltage is either MAX_CHARGING_VOLTAGE_OVERIDE or MAX_CHARGING_VOLTAGE_NOMINAL.